controllers/grid/submissions/activeSubmissions/ActiveSubmissionsListGridHandler.inc.php

```diff
@@ -100,7 +100,7 @@ protected function loadData($request, $filter) {
 		}
 
 		$nonExistingUserId = 0;
-		return $submissionDao->getActiveSubmissions($context->getId(), $title, $author, $editor, $stageId, $rangeInfo);
+		return $submissionDao->getActiveSubmissions($context->getId(), $title, $author, $editor, $stageId, $rangeInfo, true);
```

Comment:
This additional condition reminds me of the "show users with no roles in this journal" checkbox in the users area. I wonder if it's worth adding a filter checkbox for this condition?